package.json

```
@@ -6,13 +6,22 @@
   "repository": "https://github.com/AtomLinter/linter-roodi",
   "license": "MIT",
   "engines": {
-    "atom": ">=0.174.0 <2.0.0"
+    "atom": ">=v1.9.0 <2.0.0"
```

The version number must be specified as a number:
```suggestion
    "atom": ">=1.9.0 <2.0.0"
```

---

ouch